staging/src/k8s.io/apiextensions-apiserver/test/integration/basic_test.go

```diff
@@ -81,124 +81,163 @@ func TestClusterScopedCRUD(t *testing.T) {
 }
 
 func testSimpleCRUD(t *testing.T, ns string, noxuDefinition *apiextensionsv1beta1.CustomResourceDefinition, dynamicClient dynamic.Interface) {
-	noxuResourceClient := NewNamespacedCustomResourceClient(ns, dynamicClient, noxuDefinition)
-	initialList, err := noxuResourceClient.List(metav1.ListOptions{})
-	if err != nil {
-		t.Fatal(err)
-	}
-	if e, a := 0, len(initialList.Items); e != a {
-		t.Errorf("expected %v, got %v", e, a)
-	}
-	initialListTypeMeta, err := meta.TypeAccessor(initialList)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if e, a := noxuDefinition.Spec.Group+"/"+noxuDefinition.Spec.Version, initialListTypeMeta.GetAPIVersion(); e != a {
-		t.Errorf("expected %v, got %v", e, a)
-	}
-	if e, a := noxuDefinition.Spec.Names.ListKind, initialListTypeMeta.GetKind(); e != a {
-		t.Errorf("expected %v, got %v", e, a)
-	}
-
-	initialListListMeta, err := meta.ListAccessor(initialList)
-	if err != nil {
-		t.Fatal(err)
-	}
-	noxuWatch, err := noxuResourceClient.Watch(metav1.ListOptions{ResourceVersion: initialListListMeta.GetResourceVersion()})
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer noxuWatch.Stop()
-
-	createdNoxuInstance, err := instantiateCustomResource(t, testserver.NewNoxuInstance(ns, "foo"), noxuResourceClient, noxuDefinition)
-	if err != nil {
-		t.Fatalf("unable to create noxu Instance:%v", err)
-	}
-
-	select {
-	case watchEvent := <-noxuWatch.ResultChan():
-		if e, a := watch.Added, watchEvent.Type; e != a {
-			t.Errorf("expected %v, got %v", e, a)
-			break
-		}
-		createdObjectMeta, err := meta.Accessor(watchEvent.Object)
-		if err != nil {
-			t.Fatal(err)
+	noxuResourceClients := map[string]dynamic.ResourceInterface{}
+	noxuWatchs := map[string]watch.Interface{}
+	disbaledVersions := map[string]bool{}
```

Comment:
s/disbaled/disabled

Reply:
fixed

Reply:
it's still the same. maybe forgot to push? :)